examples/aggregate.php

```diff
@@ -1,7 +1,7 @@
 <?php
 declare(strict_types=1);
 
-namespace MongoDB\Examples;
+namespace MongoDB\Examples\Aggregate;
```

Comment:
Are we going to have the same problem with the [CSFLE examples](https://github.com/mongodb/mongo-php-library/tree/master/docs/examples)? Or perhaps not because those don't define any functions?

Reply:
Potentially, yes. In this particular case we won't, however I consider this an upstream issue, as blindly compiling every PHP file regardless of whether it's even available to be autoloader is an error. The PR @GromNaN created in the library excludes all files in `examples` folders from compilation, which fixes the issue for good. This is just a temporary workaround to resolve the immediate issue until it can be fixed upstream.